motelpos/MotelPos/View Controllers/Main/MainViewController+Actions.swift

```diff
@@ -103,11 +149,33 @@ extension MainViewController {
         MotelApp.current.client.initiateRecovery(referenceId, transactionType: .getLastTransaction, completion: printResult)
     }
     
+    func sanitizePrintText(_ text: String?) -> String? {
+        let boldText: String? = text?.replacingOccurrences(of: "\\n", with: "\n");
```

Comment:
Could these 3 lines simply be replaced with one call to:
replacingOccurrences(of: "\\", with: "\");

Reply:
i tried it but i doesn't work like that